x/avs/keeper/keeper.go

```diff
@@ -288,28 +288,35 @@ func (k Keeper) CreateAVSTask(ctx sdk.Context, params *types.TaskInfoParams) (ui
 
 func (k Keeper) RegisterBLSPublicKey(ctx sdk.Context, params *types.BlsParams) error {
 	// check bls signature to prevent rogue key attacks
-	sig := params.PubkeyRegistrationSignature
-	msgHash := params.PubkeyRegistrationMessageHash
+	// a message parameter to validate that this signature is intended solely for RegisterBLSPublicKey.
+	// The message should contain (ExoCore-$chain-id-$operator-address)  marker to prevent replay attacks.
+	// Looking forward to this format: "ExoCore-exocorelocalnet_232-exo13h6xg79g82e2g2vhjwg7j4r2z2hlncelwutkjr"
+	// Note that the address of the operator must be lowercase
+	expectedMessage := "BLS Signed Message-" + types.ChainIDWithoutRevision(ctx.ChainID()) + "-" + strings.ToLower(params.OperatorAddress.String())
```

Comment:
Make this a constant in x/avs/types/types.go and add newlines \n between the parts. Reuse the constant in your test